### PR TITLE
Changes from workshop walkthrough

### PR DESCRIPTION
**Pull request by juliasilge**
This PR adds some (mostly minor) changes from our workshop walkthrough today.

### CODE REVIEW

**Comment by juliasilge** [comment]
Closes #34

**Review by hfrick — APPROVED** [review]
Looks good!

File: slides/03-what-makes-a-model.qmd [hunk 1]
```diff
@@ -401,19 +414,6 @@ workflow() %>%
...
-  set_mode("regression")
-
-workflow() %>%
-  add_variables(outcomes = latency, predictors = everything()) %>%
-  add_model(tree_spec) %>%
-  fit(data = frog_train) 
-```
-
-## A model workflow `r hexes("parsnip", "workflows")`
-
```
Comment (DavisVaughan) [inline slides/03-what-makes-a-model.qmd:416]: So we removed this one but not the `add_formula(latency ~ .) %>%` one from above?

**Review by DavisVaughan — APPROVED** [review]
LGTM - The faster we can merge this the better probably. Sorry I couldn't take a look yesterday

File: slides/03-what-makes-a-model.qmd (hunk 1 above)
Comment (juliasilge) [reply slides/03-what-makes-a-model.qmd:416]: I believe Max said to keep this so we teach folks both `workflow(preprocessor, spec)` and `workflow() %>% add_formula(form) %>% add_model(spec)`.

**Pull request merged**